Dangerfile

```diff
@@ -6,8 +6,28 @@ declared_trivial = github.pr_title.include? "#trivial"
 warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
 
 # Warn when there is a big PR
-warn("Big PR") if git.lines_of_code > 500
+warn("The PR is too big. Try splitting it up") if git.lines_of_code > 20
```

Comment:
shall we do 50 lines at the start?

For the first time in the PR when the screen and UI are done, it will be bigger. is there any way we can handle it?

Reply:
yeah, I just reduced to demo how danger comments  , we can even have 100 lines to be safe. .
And it is just a warning . 
if we want to fail the build, we have to invoke `fail("Big PR")`